invenio_pages/services/service.py

```diff
@@ -7,29 +7,80 @@
 
 """Page Service API."""
 
-from flask import abort
 from invenio_access.permissions import system_identity
 from invenio_records_resources.services import RecordService
-from sqlalchemy.orm.exc import NoResultFound
+from invenio_records_resources.services.base import LinksTemplate
+from invenio_records_resources.services.base.utils import get_search_params
```

Comment:
the func has been renamed, see https://github.com/inveniosoftware/invenio-records-resources/pull/437